application/src/test/java/org/opentripplanner/service/paging/PS3_FewItinerariesOnSearchWindowLimitTest.java

```diff
@@ -65,16 +72,20 @@ class PS3_FewItinerariesOnSearchWindowLimitTest {
   static List<Arguments> testCases() {
     return List.of(
       // Itineraries depart inside search-window, step NEXT 6 times
-      //Arguments.of("08:00", DEPART_AFTER, "1 N 2 N - N - N 3 N - N"),
+      Arguments.of("08:00", DEPART_AFTER, "1 N 2 N - N - N 3 N - N"),
       // Itineraries depart inside search-window, step PREV 6 times
-      //Arguments.of("08:00+1d", DEPART_AFTER, "3 P - P - P 2 P 1 P - P"),
+      Arguments.of("08:00+1d", DEPART_AFTER, "3 P - P - P 2 P 1 P - P"),
       // Itineraries depart inside search-window, step BACK and FORTH
-      //Arguments.of("08:00", DEPART_AFTER, "1 N 2 N - P 2 N - P 2 P 1 N 2 N"),
+      Arguments.of("08:00", DEPART_AFTER, "1 N 2 N - P 2 N - P 2 P 1 N 2 N"),
```

Comment:
Why were these arguments commented out previously? I've trouble understanding what's going on in this test class in general with the pattern matching etc. Perhaps at least add a comment on the pattern for what it is used for. It's a bit difficult to read the regex and understand its usage in this context.